packages/tabs/src/uxp-tab.css

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* write uxp style overrides */
+
+:host {
+    height: calc(
+        var(--mod-tabs-item-height, var(--spectrum-tabs-item-height)) -
+            var(--mod-tabs-divider-size, var(--spectrum-tabs-divider-size))
+    );
+}
+
+#item-label {
+    margin-bottom: var(
+        --mod-tabs-bottom-to-text,
+        var(--spectrum-tabs-bottom-to-text)
+    );
+    margin-top: var(--mod-tabs-top-to-text, var(--spectrum-tabs-top-to-text));
+}
+
+:host:before {
+    height: calc(
+        100% - var(--mod-tabs-top-to-text, var(--spectrum-tabs-top-to-text))
+    );
+    width: calc(
+        100% +
+            var(
+                --mod-tabs-focus-indicator-gap,
+                var(--spectrum-tabs-focus-indicator-gap)
+            ) * 2
+    );
+    top: calc(
+        var(--mod-tabs-top-to-text, var(--spectrum-tabs-top-to-text)) / 2
+    );
+    right: calc(
+        var(
+                --mod-tabs-focus-indicator-gap,
+                var(--spectrum-tabs-focus-indicator-gap)
+            ) * -1
+    );
+    left: calc(
+        var(
+                --mod-tabs-focus-indicator-gap,
+                var(--spectrum-tabs-focus-indicator-gap)
+            ) * -1
+    );
+}
+
+::slotted([slot='icon']) {
+    width: var(--mod-tabs-icon-size, var(--spectrum-tabs-icon-size));
+    margin-top: var(--mod-tabs-top-to-icon, var(--spectrum-tabs-top-to-icon));
+}
+
+[name='icon'] + #item-label {
+    margin-left: var(
+        --mod-tabs-icon-to-text,
+        var(--spectrum-tabs-icon-to-text)
+    );
+}
+
+:host([disabled]) {
+    color: var(
```

Comment:
what is the issue with disabled color state in existing styles?

Reply:
```
::slotted(:not([slot])):focus, ::slotted(:not([slot])):focus-visible {
    color: var(--highcontrast-tabs-color-key-focus, var(--mod-tabs-color-key-focus, var(--spectrum-tabs-color-key-focus))); 
}
```

Above style is conflicting for the disabled state style, so this is required to increase the specificity and hence this gets picked

Reply:
does SWC not have disabled styles?

Reply:
its there but in UXP somehow, the above style gets the higher specificity hence i had to add disabled styles explicitly with important keyword